client/src/App.js

```diff
@@ -25,10 +25,13 @@ const App: () => React$Node = () => {
           <PickupDetail />
         </Route>
         <Route exact path="/manage-customer">
-          <ManageCustomer />
+          <CustomerList />
         </Route>
         <Route exact path="/modify-a-customer">
-          <ModifyCustomer />
+          <CustomerDetails />
```

Comment:
is there still a `/modify-a-customer` page based on the new behaviours?

Reply:
No. We don't use `/modify-a-customer` now, since we only use `create-a-customer` for `<CustomerCreate />` component.

Reply:
What I am implying is that we should remove this route if this is no longer in use.